### ISSUE TITLE
Vote positions not announced correctly by all screen readers

### ISSUE THREAD

**Issue by tillprochaska** [HowTheyVote/howtheyvote#1021]
The `span` element should have an extra `aria-label="against"` attribute as some screen readers ignore empty elements (even though the element has a `title` attribute).

```html
<span class="thumb thumb--against thumb--circle " title="against"><svg aria-hidden="true"><use href="/static/icons.svg#thumb"></use></svg></span>
```
